data/systemd/Makefile

```diff
@@ -0,0 +1,28 @@
+SNAPD_ENVIRONMENT_FILE := /etc/environment
+SNAP_MOUNTDIR := /snap
+BINDIR := /usr/bin
+LIBEXECDIR := /usr/lib
+SYSTEMDSYSTEMUNITDIR := /lib/systemd/system
+
+all:
```

Comment:
I was thinking about if this could be made more make-ish. What do you think about something like the bits below:
```
SYSTEMD_UNITS := $(patsubst %.service.in,%.service,$(wildcard *.service.in))

%.service: %.service.in
	cat $< | \
		sed s:@libexecdir@:${LIBEXECDIR}:g | \
		sed s:@SNAPD_ENVIRONMENT_FILE@:${SNAPD_ENVIRONMENT_FILE}:g | \
		sed s:@bindir@:${BINDIR}:g | \
		sed s:@SNAP_MOUNTDIR@:${SNAP_MOUNTDIR}:g  | \
	cat > $@


install: $(SYSTEMD_UNITS)
```
?